### ISSUE TITLE
Interop with GPU compute kernels

### ISSUE THREAD

**Issue by kwryankrattiger** [ECP-VeloC/VELOC#41]
Is there a plan for veloc to support a direct HIP/CUDA interface?

**Comment by bnicolae** [comment]
Yes, we have an experimental version. We will open it for general use soon

**Comment by kwryankrattiger** [comment]
Do you think this will be available my Feb 15 next year?

Will it be ready for both CUDA and HIP, or Kokkos based to cover all the bases?

**Comment by yulingao** [comment]
@bnicolae Look forward to your CUDA support update!
